sunpy/lightcurve/sources/let.py

```diff
@@ -0,0 +1,97 @@
+from datetime import timedelta,datetime
+from astropy.io import ascii
+from astropy.table import Table, Column
+
+def _parse_txt(filepath):
+    """
+    Parses a STEREO LET file from
+    http://www.srl.caltech.edu/STEREO/Public/LET_public.html
+
+    """
+    #Reading in Header  
+
+    type_of_data = ''
+    header = []
+    flag1 = -1
+    flag2 = -1
```

Comment:
I would not use `flags` but a dictionary (if they refer to more than one thing) and `True` or `False` to test them